src/pages/Main/Main.js

```diff
@@ -1,5 +1,267 @@
+
+  return (
+    <Wrapper>
+      <div>
+        <MenuContainer>
+          <MenuTab>
+            <MenuList onMouseLeave={() => setIsShowed(false)}>
+              {CATEGORY_ARR.map((category, idx) => {
+                return (
+                  <li
+                    key={category + idx}
+                    onMouseOver={() => {
+                      clickHandler(idx + 1);
+                      setIsShowed(true);
+                    }}
+                  >
+                    {category}
+                  </li>
+                );
+              })}
+            </MenuList>
+            <Contents
+              className={isShowed && 'show'}
+              onMouseOver={() => setIsShowed(true)}
+              onMouseLeave={() => setIsShowed(false)}
+            >
+              {MAPPING_OBJ[currentId].map((item, idx) => (
+                <li
+                  key={idx}
+                  id={4 * (currentId - 1) + parseInt(idx) + 1}
+                  onClick={goToCategoryList}
+                >
+                  {item}
+                </li>
+              ))}
+            </Contents>
+          </MenuTab>
+          <MainSlider />
+        </MenuContainer>
+
+        <SubMenu>
+          <SubList id="1" onClick={goToTypeList}>
+            <i id="1" className="fas fa-map-marker-alt" />
+            오프라인
+          </SubList>
+          <SubList id="2" onClick={goToTypeList}>
+            <i id="2" className="fas fa-video" />
+            VOD
+          </SubList>
+          <SubList id="3" onClick={goToTypeList}>
+            <i id="3" className="fas fa-smile" />
+            신규오픈
+          </SubList>
+        </SubMenu>
+        <MainContainer>
+          <LectureContainer>
+            <h3>지금! 뜨고 있는 클래스</h3>
+            <LectureSlide />
+            <h3>당장 들어야 할 오프라인 클래스</h3>
+            <LectureSlide />
+            <h3>뿌잉 VOD, 같이하면 더 쉬우니까</h3>
+            <LectureSlide />
+          </LectureContainer>
+          <ProfileContainer>
+            <Profile>
+              <h3 className="userId">효정님</h3>
+              <h3>반가워요</h3>
+              <p>
+                포인트 <span>0</span>p
+              </p>
+              <button>
+                <i className="fas fa-heart" /> 찜
+              </button>
+            </Profile>
+            <Search>
+              <h3>핫 트렌드</h3>
+              <span>#강의 이름</span>
+              <span>#강의 이름</span>
+              <span>#강의 이름</span>
+              <span>#강의 이름</span>
+              <span>#강의 이름</span>
+              <span>#강의 이름</span>
+            </Search>
+          </ProfileContainer>
+        </MainContainer>
+      </div>
+    </Wrapper>
+  );
 };
 
 export default Main;
+
+const MAPPING_OBJ = {
+  1: ['국내', '일본', '유럽', '미국'],
+  2: ['한식', '중식', '일식', '커피 · 디저트'],
+  3: ['드로잉', '미술', '글쓰기', '사진'],
+  4: ['러닝', '등산', '축구', '요가'],
+};
+
+const CATEGORY_ARR = ['여행', '요리', '드로잉 · 사진', '피트니스'];
+
+const Wrapper = styled.div`
+  ${props => props.theme.wrapper()}
```

Comment:
```suggestion
  ${({theme}) => theme.wrapper()}
```
- 구조분해할당을 통해서 좀 더 간결하게 표현할 수 있습니다!

Reply:
네 수정하였습니다!